include/RAJA/operators.hxx

```diff
@@ -245,11 +245,11 @@ template <typename T>
 RAJA_HOST_DEVICE constexpr T min()
 {
   return 0;
-};
+}
 template <>
 RAJA_HOST_DEVICE constexpr int16_t min<int16_t>()
 {
-  return 0xFFFF;
+  return -32768;
```

Comment:
Yup, you can push more commits to the branch and they will show up. We're really grateful for the PR!

Personally, I like the `numeric_limits` approach.